server/lib/report-manager.js

```diff
@@ -110,7 +112,7 @@ class ReportManager
         content += '\n';
 
         for (let account of accounts) {
-            let lastCheck = moment(account.lastCheck).format('DD/MM/YYYY');
+            let lastCheck = moment(account.lastCheck).locale(locale).format('L');
```

Comment:
Any chance to hoist this line into an helper function, in the shared helpers? Specifically, we shouldn't need to load the locale each time...